ts/Extensions/Drilldown.ts

```diff
@@ -622,7 +624,22 @@ defaultOptions.drilldown = {
              */
             y: 10
         }
-    }
+    },
+
+    /**
+     * Enable or disable the zooming into the region of clicked map point you
```

Comment:
... disable zooming into a region of the clicket map point ...

Reply:
If `mapZooming` is false the drilldown/drillup animations only fade in/fade out without zooming to a specific MapPoint.

Reply:
Okay, we should explain that in the doclet description.